src/morse/index.js

```diff
@@ -58,6 +58,42 @@ const MORSE_CODE = {
 
 Object.freeze(MORSE_CODE);
 
-function morse(text) {}
+function morse(text) {
+  if (typeof text !== 'string') {
+    throw ('Please provide a morse string');
```

Comment:
Prefer to `throw Error(...)` rather than strings. Helps preserve the stack trace for debugging.